x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/ForecastIT.java

```diff
@@ -276,6 +278,55 @@ public void testOverflowToDisk() throws Exception {
 
     }
 
+    public void testDelete() throws Exception {
```

Comment:
We should also add a YML test in `forecast.yml`. Ping me if you need a tutorial on how those work.